backend/lib/services/terminals/index.js

```diff
@@ -333,27 +333,23 @@ function getHostCluster ({ user, namespace, name, target, body, shootResource })
   return getShootHostCluster(client, { namespace, name, target, body, shootResource })
 }
 
-async function createTerminal ({ user, namespace, name, target, hostCluster, targetCluster, body }) {
+async function createTerminal ({ user, namespace, target, hostCluster, targetCluster, body }) {
   const client = user.client
   const isAdmin = user.isAdmin
   const { identifier } = body
   const containerImage = getContainerImage({ isAdmin, preferredContainerImage: hostCluster.containerImage })
+  const containerArgs = hostCluster.containerArgs
```

Comment:
Use hierarchy for hierarchy:
```js
{ 
  container: {
    preferredImage': 'prefName', 
    image: 'name',
    args: []
  }
}
```

Reply:
this shouldn't be part of this PR, will remove it again. But I will consider it for the upcoming PR